cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -102,24 +119,33 @@ Now we run the same queries again:
 MATCH (n:User) WHERE n.name CONTAINS 'ndy' RETURN n.name;
 ----
 
-This query will operate exactly as before, returning the same single result, because nothing in this query relates to the denylisted property.
+This query will operate exactly as before, returning the same two results, because nothing in this query relates to the blacklisted property.
```

Comment:
```suggestion
This query operates exactly as before, returning the same two results, because nothing in this query relates to the denylisted property.
```